desc/compute/_field.py

```diff
@@ -2331,21 +2612,23 @@ def _B_rms(params, transforms, profiles, data, **kwargs):
     transforms={"grid": []},
     profiles=[],
     coordinates="r",
-    data=["sqrt(g)", "|B|", "V_r(r)"],
+    data=["sqrt(g)", "|B|"],
+    axis_limit_data=["sqrt(g)_r"],
 )
 def _B_fsa(params, transforms, profiles, data, **kwargs):
     data["<|B|>"] = surface_averages(
         transforms["grid"],
         data["|B|"],
-        data["sqrt(g)"],
-        denominator=data["V_r(r)"],
+        sqrt_g=transforms["grid"].replace_at_axis(
```

Comment:
No because the integrand of the numerator is $\lvert B \rvert \partial_{\rho} \sqrt{g} + \sqrt{g} \partial_{\rho} \lvert B \rvert$ and the second term is 0 at the axis because $\partial_{\rho} \lvert B \rvert$ is finite.